packages/dx-react-chart/src/templates/tooltip/target.jsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+
+export class Target extends React.PureComponent {
+  render() {
+    const {
+      d, x, y, componentRef,
+    } = this.props;
+    return (
+      <path transform={`translate(${x} ${y})`} fill="none" ref={componentRef} d={d} />
+    );
+  }
+}
+
+Target.propTypes = {
+  x: PropTypes.number,
```

Comment:
Why are `x`, `y` and `d` not required?

Reply:
If there is no target that parameters are absent

Reply:
If there is no target should `Target` component be rendered at all?